### PR TITLE
Bluespace Event: Science Disaster

### PR DESCRIPTION
**Pull request by Helicaste**
## About the PR

Adds an R&D-focused event POI.

No serious spoilers for what's onboard, but it is a High Hazard event that should be challenged by experienced dungeon delvers. I also do not recommend approaching any way except the front door.

Generally, you will find all manner of R&D items, alongside a few of the more... special things in life. However, beware the vault. It is not safe.

## Why / Balance

:)

## Technical details

.yml changes

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

:cl:
- add: Added the Bluespace Event: Science Disaster. Be on the lookout for a rich vault full of... carp?

### CODE REVIEW

File: Resources/Prototypes/_NF/Events/events.yml [hunk 1]
```diff
@@ -141,3 +141,22 @@
     maxDuration: 1200
   - type: BluespaceErrorRule
     gridPath: /Maps/Bluespace/cargoniaship.yml
+
+    - type: entity
```
Comment (dvir001) [inline Resources/Prototypes/_NF/Events/events.yml:145]: Your .yml format is broken here.

**Comment by FoxxoTrystan** [comment]
I see a few issues and recomendation:

Issues:
* WarpPoint is unamed.
* The RatKing Spawn is a ghost role. (only a ghost and take over it)
* All the walls are Indestructible Plastitanum walls, depands ofc how the design is made but the interior also witch is annoying for players.
* .yml format broken.

Recomendation:
* Use the Unsual Asteroid Annoncement and asteroid name (so peoples does not know what is it about until they see it in sensors)
* Avoid GasMiners (gasminers personnally should be avoided in Ships/Bluespace events unless POI.
* Add some decals, there is no decals currently and the floor seem way too clean, well the lab do look clean.
* Provide more information in the PR for next time.
* Duration/Max Duration should be changed pehaps.

Anywho good work on the map, in my opinion the loot is "too great" and to easy but i have not tested it, i do suggest tho that the asteroid falls under the "Asteroid" category so peoples will not know what its is until they go to it.

**Comment by Helicaste** [comment]
> I see a few issues and recomendation:
> 
> Issues:
> 
> * WarpPoint is unamed.
> * The RatKing Spawn is a ghost role. (only a ghost and take over it)
> * All the walls are Indestructible Plastitanum walls, depands ofc how the design is made but the interior also witch is annoying for players.
> * .yml format broken.
> 
> Recomendation:
> 
> * Use the Unsual Asteroid Annoncement and asteroid name (so peoples does not know what is it about until they see it in sensors)
> * Avoid GasMiners (gasminers personnally should be avoided in Ships/Bluespace events unless POI.
> * Add some decals, there is no decals currently and the floor seem way too clean, well the lab do look clean.
> * Provide more information in the PR for next time.
> * Duration/Max Duration should be changed pehaps.
> 
> Anywho good work on the map, in my opinion the loot is "too great" and to easy but i have not tested it, i do suggest tho that the asteroid falls under the "Asteroid" category so peoples will not know what its is until they go to it.

I like the idea that it uses the same announcements as the Unusual Asteroid, the loot _is_ quite strong but there's a couple tricks to the event that aren't immediately obvious from ghosting over it.

I did want to avoid listing anything specific as being in the event's rewards or challenges though, which is why it's low detail as there isn't _that_ much to say other than "It's hard and rewarding".

Broken .yml format was something I fixed clientside and I guess I forgot to fix it in the PR, easy to do though. Same with the announcement.

Gas Miners and Decals are something I can do with a second pass alongside updating the warp point and adding actual Research Point disks (because I forgor), though it was somewhat of an intentional design decision to include the miners as the event's intended to be on the longer side, and allow players to "siphon" gas if they're so inclined. Shouldn't be an issue to remove them though.

Same kinda goes for the Plastitanium walls, an issue I noticed on the other asteroid is it's not that hard to bypass some of the challenges by breaking specific walls, and this is definitely intended to be a more maze-like structure. Most of the walls ended up being one-tile thick, so i'm somewhat against making them destructible in most places. Not all walls are unbreakable though.

**Comment by Cheackraze** [comment]
ill have to load this up to review it properly. Gas miners I could possibly accept on a bluespace event like this, depending on how they are implemented. Also, would probably want their output nerfed from the default low-flow kind too.  As for indestructible walls, they should be used sparingly, and your trap design really should just incorporate or take into account the fact that players *can* and likely *will* go through them.  Its not necessarily a bad thing, a big part of this game is using all the tools available to you, including your knowledge of game mechanics, to find new and emergent ways of dealing with situations.

**Comment by dvir001** [comment]
Please reopen this if you are still working on this event
Also keep in mind new folders to save the files into are now under _NF

**Pull request closed without merging**